pkgs/by-name/fe/feluda/package.nix

```diff
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "feluda";
+  version = "1.3.0";
+
+  src = fetchFromGitHub {
+    owner = "anistark";
+    repo = "feluda";
+    tag = "1.3.0";
+    hash = "sha256-";
```

Comment:
doesn't work like that. You have to put the hash too. sha256- is just the prefix